etc/collectd.d/nginx.conf

```diff
@@ -5,5 +5,5 @@ LoadPlugin nginx
 <Plugin "nginx">
     # When using non-standard Nginx configurations, replace the below with
     #URL "http://NGINX_HOST:NGINX_PORT/nginx_status"
-    URL "http://localhost/nginx_status"
+    URL "http://local-stackdriver-agent.stackdriver.com:80/nginx_status"
```

Comment:
Running `dig local-stackdriver-agent.stackdriver.com` contains the following:
```
;; ADDITIONAL SECTION:
.			0	IN	TXT	"Error" "Answer blocked by policy."
```
So I'd say we're good.